src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1194,6 +1195,57 @@ IF(PLUS_USE_CAPISTRANO_VIDEO)
 ENDIF()
 
 # --------------------------------------------------------------------------
+# WinProbe support
+IF(PLUS_USE_WINPROBE_VIDEO)
+  SET(WinProbe_SRCS
+    WinProbe/vtkPlusWinProbeVideoSource.cxx
+    )
+  IF(MSVC OR ${CMAKE_GENERATOR} MATCHES "Xcode")
+    SET(WinProbe_HDRS
+      WinProbe/vtkPlusWinProbeVideoSource.h
+      )
+  ENDIF()
+  
+  SET(PlusDataCollection_HDRS ${PlusDataCollection_HDRS}
+    ${WinProbe_HDRS}
+    )
+
+  SET(PlusDataCollection_SRCS ${PlusDataCollection_SRCS}
+    ${WinProbe_SRCS}
+    )
+
+  SET(PlusDataCollection_INCLUDE_DIRS ${PlusDataCollection_INCLUDE_DIRS}     
+    ${WINPROBESDK_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}/WinProbe
+    )
+
+  FILE(GLOB WinProbeXMLFiles "${WINPROBESDK_DIR}/xml/*.xml")
```

Comment:
There are many drawbacks of globbing (look at CMake documentation for details), so although it's slightly inconvenient, it's usually still better to enumerate all files by hand.
Globbing files from a third-party directory is somewhat less of an issue, so if you really want to do it, then I think it's acceptable (we went back and forth on this for other SDK files and ended up globbing for some of them).